packages/adblocker/test/resources.test.ts

```diff
@@ -272,6 +280,12 @@ describe('#Resources', function () {
     it('includes setup for scritplet globals', function () {
       expect(resources.getScriptlet('a')).to.include('var scriptletGlobals = {};');
     });
+
+    it('allows resources surrogate', function () {
+      expect(resources.getScriptlet('surrogate'))
```

Comment:
are those things always called without `.js` ?

Reply:
Yes, resources surrogates are always called without suffix or file extension. Also, uBo gradually plan to drop calling scriptlets with suffix or file extension in the future.